GetServiceMetadata/__tests__/handler.test.ts

```diff
@@ -23,15 +23,18 @@ import { ServiceScopeEnum } from "@pagopa/io-functions-commons/dist/generated/de
 import * as TE from "fp-ts/lib/TaskEither";
 import { aCosmosResourceMetadata } from "../../__mocks__/mocks";
 import { GetServiceMetadataHandler } from "../handler";
+import { StandardServiceCategoryEnum } from "@pagopa/io-functions-commons/dist/generated/definitions/StandardServiceCategory";
 
 afterEach(() => {
   jest.resetAllMocks();
   jest.restoreAllMocks();
 });
 
 const anOrganizationFiscalCode = "01234567890" as OrganizationFiscalCode;
-const aServiceMetadata = {
-  scope: ServiceScopeEnum.NATIONAL
+const aServiceMetadata: ServiceMetadata = {
+  scope: ServiceScopeEnum.NATIONAL,
+  category: StandardServiceCategoryEnum.STANDARD,
+  customSpecialFlow: undefined
```

Comment:
Why are these changes here? Are they correlated with Node18 update or just "boyscout rule" to keep things clean and updated?

Reply:
After the `yarn.lock` regeneration `tsc` was complaining where this `aServiceMetadata` was used cause it is not a valid `ServiceMetadata` without those 2 fields